### PR TITLE
Refactor attrs of MessageListView

### PR DESCRIPTION
**Pull request by samiuelson**
This PR:
1. Renames a few attrs set names that didn't have "stream" prefixes. 
2. Introduces 2 new attrs for `MessageListView`: `streamEditMessageActionEnabled` and `streamDeleteMessageActionEnabled`

### Checklist`

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [x] Changelog updated with client-facing changes
~- [ ] New code is covered by unit tests~
~- [ ] Comparison screenshots added for visual changes~
- [x] Reviewers added

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 - Use custom `StreamFileProvider` instead of androidx `FileProvider` to avoid conflicts
 - Add `ChatClient::setGuestUser` method to login as a guest user
 - Make `MessageListItemViewHolder` public and open, to allow customization by overriding the `bind` method
+- Add new attrs to `MessageListView`: `streamDeleteMessageActionEnabled`, `streamEditMessageActionEnabled`
```
Comment (JcMinarro) [inline CHANGELOG.md:7]: Wrong file.
We are using `UNRELEASED_CHANGELOG.md` file now

File: stream-chat-android/src/main/java/com/getstream/sdk/chat/view/dialog/MessageMoreActionDialog.kt [hunk 2]
```diff
@@ -63,6 +63,8 @@ public class MessageMoreActionDialog(
                 dismiss()
             }
             binding.flagMessageButton.isVisible = style.flagMessageActionEnabled
+            binding.editMessageButton.isVisible = style.editMessageActionEnabled
+            binding.deleteMessageButton.isVisible = style.deleteMessageActionEnabled
```
Comment (zsmb13) [inline stream-chat-android/src/main/java/com/getstream/sdk/chat/view/dialog/MessageMoreActionDialog.kt:67]: Why are we adding this to the branch where the message is not by the current user?

File: stream-chat-android/src/main/res/values/attrs.xml [hunk 3]
```diff
@@ -178,7 +178,7 @@
         <attr name="streamSendAlsoToChannel" format="color|reference" />
     </declare-styleable>
     <!--MessageListView-->
-    <declare-styleable name="MessageListView">
+    <declare-styleable name="StreamMessageListView">
```
Comment (zsmb13) [inline stream-chat-android/src/main/res/values/attrs.xml:181]: Looks like the layout editor relies on the name of the styleable matching the name of the View to show suggestions. I don't get any autocompletion results when I try to add attributes (for example, the newly added ones) to the `MessageListView` in `fragment_channel.xml` after this change.

I think we should remove prefixes from all of our styleables, as our Views don't have them, and then suppress the warning about their names, if possible.

**Review by zsmb13 — CHANGES_REQUESTED** [review]

File: stream-chat-android/src/main/res/values/attrs.xml (hunk 3 above)
Comment (samiuelson) [reply stream-chat-android/src/main/res/values/attrs.xml]: It's reverted now. Let's clean it up in another PR.

File: stream-chat-android/src/main/java/com/getstream/sdk/chat/view/dialog/MessageMoreActionDialog.kt (hunk 2 above)
Comment (samiuelson) [reply stream-chat-android/src/main/java/com/getstream/sdk/chat/view/dialog/MessageMoreActionDialog.kt]: Fixed

**Review by zsmb13 — APPROVED** [review]

**Pull request merged**